docs/concepts/move.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Move"
+linkTitle: "Move"
+weight: 1
+---
+
+## Moving Ondat volume instances
+
+A volume instance (replica or primary) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.
+
+It allows to manually re-balance a cluster, reduce load on over-loaded nodes, etc.
```

Comment:
~It allows to manually re-balance a cluster~
It can be used to manually re-balance the cluster

Reply:
Done